packages/rocketchat-videobridge/client/views/videoFlexTab.js

```diff
@@ -0,0 +1,64 @@
+/* globals JitsiMeetExternalAPI */
+
+Template.videoFlexTab.helpers({
+
+});
+
+Template.videoFlexTab.onCreated(function() {
+	let rid = this.data && this.data.rid;
+	let api;
+
+	let timeOut = null;
+
+	this.timeout = null;
+	this.autorun(() => {
+		if (RocketChat.settings.get('Jitsi_Enabled')) {
+			if (RocketChat.TabBar.getTemplate() === 'videoFlexTab') {
+				if (RocketChat.TabBar.isFlexOpen()) {
+					RocketChat.TabBar.updateButton('video', { class: 'red' });
+				} else {
+					RocketChat.TabBar.updateButton('video', { class: '' });
+
+					if (timeOut) {
+						console.log(timeOut);
+						api.dispose();
+						clearInterval(timeOut);
+					}
+				}
+			}
+		}
+	});
+
+	// Opening a PR so we can do this via the https://meet.jit.si/external_api.js
+	$.getScript('https://cdn.rawgit.com/geekgonecrazy/jitsi-meet/master/external_api.js')
+		.done(function() {
+			var domain = RocketChat.settings.get('Jitsi_Domain');
+			var room = CryptoJS.MD5(RocketChat.settings.get('uniqueID') + rid).toString();
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [A function with a name starting with an uppercase letter should only be used as a constructor.](https://www.codacy.com/app/RocketChat/Rocket-Chat/file/2717439985/issues/source?bid=3199634&fileBranchId=3342079#l36)